dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Audacia.CodeAnalysis.Analyzers.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <Description>This package supports C# 12 and .NET 8 as a minimum.</Description>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
I don't think the `TargetFramework` and `LangVersion` of this project need to change. This project itself doesn't need to be .NET 8 - the point is that it can analyze other code that is .NET 8+.

Reply:
Ah my apologies! I had misunderstood the AC. I'll change back :)

Reply:
I have reverted those changes and updated the description slightly to hopefully make it clearer

Reply:
I have reverted those changes and updated the description slightly to hopefully make it clearer